app.js

```diff
@@ -185,6 +185,7 @@ function setInitialValues() {
       language: 'en',
       sound: 'default',
       muted: false,
+      notification: true,
```

Comment:
I think this is not necessary. The notification should always be on, don't you think?